### ISSUE TITLE
Add menu item to 'Locate In Tree' when MB3 on request tab

### ISSUE THREAD

**Issue by SegarJJ** [a85/POSTMan-Chrome-Extension#7351]
I have a lot of request tabs open in my workspace, many of them are from some of the different collections in the workspace.  I have changes in many of these tabs, but I am not sure to save my changes on that request or not... it depends on which collection it was in.   As a result, I often stand to loose work because I am unsure where that request was opened from.

To help, I propose to get a menu option when MB3 from the request tab, which would open the collection tree on the left and highlight the request I have selected?

![image](https://user-images.githubusercontent.com/23700786/65235782-98ecf280-dad7-11e9-9ffd-191f0b8afb4d.png)

**Comment by andrkhr** [comment]
Yes, this is often very necessary. Move to the area where the open request is located.

**Comment by xsimo** [comment]
+1, This is a must !

**Comment by kool79** [comment]
It is required when I do search over collections. From Search Results>Collections I can "open in builder" any request. But then I cannot navigate to that request in Tree.
Also I cannot "open in Tree" from search results. There is a path to collection here, but:
 -  it is truncated and does not allow to search for item
 - even if it is short and readable it is too annoying to expand and search every item manually in collection with 300+ requests. Especially for find+fix+replace case
